Docker/launch_ros.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+. /opt/ros/${ROS_DISTRO}/setup.sh
```

Comment:
I would either keep the shebang line pointing to `sh` or call `bash` scripts whenever possible.